### PR TITLE
Wiring Editor: upgrade missing components

### LINKED ISSUE #181: Wiring Editor: missing components are not upgraded/downgraded correctly
There is a issue when you try to upgrade a missing component. Such component is upgraded successfully but the message of missing references is still displayed.

### PR DESCRIPTION
**Pull request by jpajuelo**
This pull-request fixes #181 

The issue was because of the missing components could not change their missing status. When a missing component was created and then was updated, such component loaded the configuration of that version properly but kept the missing status.

### CODE REVIEW

File: src/wirecloud/platform/static/js/wirecloud/wiring/Operator.js [hunk 1]
```diff
@@ -116,7 +116,10 @@
             build_endpoints.call(this);
             build_prefs.call(this, businessInfo.preferences);
 
-            this.logManager.log(utils.gettext("The operator was created successfully."), Wirecloud.constants.LOGGING.INFO_MSG);
+            this.logManager.log(utils.gettext("The operator was created successfully."), {
+                level: Wirecloud.constants.LOGGING.INFO_MSG,
+                console: false
```
Comment (aarranz) [inline src/wirecloud/platform/static/js/wirecloud/wiring/Operator.js]: do this console option exist?

Comment (jpajuelo) [inline src/wirecloud/platform/static/js/wirecloud/wiring/Operator.js]: Yes, it does. Check it out!
https://github.com/Wirecloud/wirecloud/blob/develop/src/wirecloud/platform/static/js/wirecloud/LogManager.js#L80

File: src/wirecloud/platform/static/js/wirecloud/LogManager.js [hunk 2]
```diff
@@ -19,198 +19,204 @@
  *
  */
 
-/*global console, gettext, interpolate, Wirecloud*/
+/* global StyledElements, Wirecloud, console */
```
Comment (aarranz) [inline src/wirecloud/platform/static/js/wirecloud/LogManager.js]: globals

**Comment by aarranz** [comment]
This pull request should be split into two pull request, one for refactoring the log managers and other for fixing the error when upgrading the missing components.

The log manager refactoring part of this pull request is partial and need to be fixed before merging.

**Comment by aarranz** [comment]
Applied manually

**Pull request closed without merging**